modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
```

Comment:
> ... and will still require internet access to use your cluster

Can you unpack this?  I expected access to cloud APIs to be all you'd need.

Reply:
@wking, I might have misunderstood, but don't you need internet access to use the cloud APIs?

Reply:
AWS populates routes to some of its endpoints, so you can reach them from inside the VPC, even if you've blocked all "external" traffic.  Or something like that.  Some things (like IAM) don't get this treatment, so the cred operator would want a way to get out to AWS's external IAM endpoint.